Bottom: f978839f80ab6eca939235df2e1f187c4deba39b
Top:    06700a3863ec7546d97bfeee3087c6157228a42a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:46:26 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f4688ac..1f355ff 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2685,7 +2685,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
